tests/unit/amo/components/TestFooter.js

```diff
@@ -65,8 +65,6 @@ describe(__filename, () => {
     expect(root.find('.Footer-lockwise-link')).toHaveProp(
       'href',
       `https://www.mozilla.org/firefox/lockwise/${makeQueryStringWithUTM({
-        utm_source: 'addons.mozilla.org',
-        utm_medium: 'referral',
```

Comment:
I don't understand why these aren't included any longer?

Reply:
Same as above: we don't need to specify the values because they are default values.